packages/core/src/i18n/screens/common/en-US.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import {Common} from './model';
+
+export const common: Common = {
+  copyright: '© {{currentYear}} WSO2 LLC Test.',
```

Comment:
What is the reason to add `Test` here?

Reply:
Removed with 556a906dd5fdf9a0c832e492b844d1d78e6713db